pkg/handlers/generic/clusterconfig/variables_test.go

```diff
@@ -19,8 +20,31 @@ func TestVariableValidation(t *testing.T) {
 		ptr.To(v1alpha1.GenericClusterConfig{}.VariableSchema()),
 		false,
 		NewVariable,
+		// CNI
 		capitest.VariableTestDef{
-			Name: "valid proxy config only",
+			Name: "CNI: set with valid provider",
```

Comment:
Can we split these variables tests into separate test files per variable property? Otherwise I think this is going to get difficult to maintain and ensure we have coverage of variable definitions.